### DISCUSSION TITLE
Speed and RAM usage. Non-default are valid options?

### DISCUSSION THREAD

**Discussion by JM-Bader** [vdemichev/DiaNN#418] [Q&A] [answered]
The default is 
![image](https://user-images.githubusercontent.com/98888664/175348405-84f9251f-62ad-4450-bff3-d43893adfb05.png)

I would like to make my search faster, so there also is 
- low RAM & high Speed
- Ultra-fast

Are these valid options ? Or experimental etc. so you would not recommend them?
What shall I expect if I use them ? 
- estimated XY% fewer IDs ?
- worse quantification ? is there an estimate?  
- how much faster does it get?

**Comment by vdemichev** [comment, marked as answer]
Ultra-fast is very fast (~5x speed up lib-free) but ID numbers are significantly lower and FDR is a bit relaxed. They are valid.

**Reply by JM-Bader** [reply]
how about 'low RAM & high Speed'?

**Reply by vdemichev** [reply]
Minimal loss of IDs. Can point the mouse at the option, and it will show a tooltip with description

**Reply by JM-Bader** [reply]
thanks, wasn't aware actually!
It says, only recommended when analysis of lib-free takes too long. 
in my case, analysis with library takes too long (thousands of samples) - so I guess it still is a valid option. Correct?

**Reply by vdemichev** [reply]
Well, yes, but IDs will be lower.
